docs/site_organization/organizing_the_site.md

```diff
@@ -72,6 +72,21 @@ to have relations only to Content items of a specific, or any Content Type.
     You can delete a Content Type only when there are no Content items that belong to it.
     This also includes Content items in the Trash.
 
+### Setting relation starting location
+
+[Relation Field types](../creating_content_basic.md) allow to configure the location in the Content tree which opens in the Universal Discovery Widget (UDW).
+
+To set the starting location for the relation, perform the following steps. 
+Go to **Content Types** and edit the respective Content Type.
+Next, go to **Field definitions**. In the `objectrelation` Field, select from the available options:
```

Comment:
```suggestion

1. Go to **Content Types** and edit the respective Content Type.
2. Go to **Field definitions** and either select or add the `objectrelation` Field
3. Under **Select starting location**, select from the available options:
```

Reply:
I follow the pattern: one sentence, one procedure. Using 'or' is not the best word as UX is concerned.

Reply:
The point is that the `objectrelation` field could be missing (like when you create a new content type or develop an existing one) and the admin might want to add it. This is not UX, this is how it may look in the product and in my opinion we should account for differing conditions.